src/commons/config/configuration.py

```diff
@@ -38,6 +38,10 @@ def backup_settings_custom_project_list(self):
     def backup_project_id(self):
         return self.__project_config['project_settings'].get('backup_project_id')
 
+    @property
+    def metadata_storage_project_id(self):
+        return self.__project_config['project_settings'].get('metadata_storage_project_id')
```

Comment:
we have unit tests for configuration, please add test for new configuration entry

Reply:
Done